### PR TITLE
docs(svelte): Adds setup information for testing-library-svelte

### PR DESCRIPTION
**Pull request by timvancleef**
Adds a missing jest transform config setting when using es6 modules.

### CODE REVIEW

File: docs/svelte-testing-library/setup.md [hunk 1]
```diff
@@ -47,25 +47,36 @@ with any testing framework and runner you're comfortable with.
     }
     ```
 
-5.  This is optional but it is recommended, you can install
+5.  If you are using es6 modules in your project you have to add jest's babel
```
Comment (m98) [inline docs/svelte-testing-library/setup.md:50]: would be nicer to say "ES6" and "Jest" instead of "es6" and "jest"

**Review by m98 — CHANGES_REQUESTED** [review]
I don't have any comment on if this change is necessary or not. But I only found two minor issues which I liked to mention.

File: docs/svelte-testing-library/setup.md (hunk 1 above)
Comment (timvancleef) [reply docs/svelte-testing-library/setup.md]: Agreed and done 🙂

**Review by m98 — COMMENTED** [review]
I have not any other feedback. 

@mihar-22 and @benmonro do you have any feedback?

**Comment by mihar-22** [comment]
Mmm I'm not really sure because there is a section right now for people who need Babel: https://testing-library.com/docs/svelte-testing-library/setup#babel--preprocessor. I guess the question is whether we should add it by default... and I think we should since I can't imagine anyone testing Svelte with Jest is not using ES modules. A few recommendations:

- Add an install instruction to step 5: `npm install babel-jest -D`
- Add a default `.babelrc` setup to step 5:

```json
{
  "presets": [["@babel/preset-env", { "targets": { "node": "current" } }]]
}
```
- Remove Babel from the bottom of the setup docs and just leave all the information about preprocessors.

**Comment by timvancleef** [comment]
Hi @mihar-22, @m98 do you have any further opinions on this PR?

**Comment by mihar-22** [comment]
Sorry for the delayed response @timvancleef, I have nothing to add looks good to me.

**Pull request merged**